interface/billing/era_payments.php

```diff
@@ -75,214 +75,284 @@ function era_callback(&$out)
             $alertmsg .=  xl("and processed.").' ';
         } else {
             $alertmsg .=  xl("but not yet processed.").' ';
-        }
-
-        ;
+        };
     }
-
     rename($tmp_name, $erafullname);
 } // End 835 upload
 //===============================================================================
 
 //===============================================================================
 ?>
+<!DOCTYPE html>
 <html>
 <head>
-<?php if (function_exists('html_header_show')) {
-    html_header_show();
-} ?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker-2-5-4/build/jquery.datetimepicker.min.css">
-
-<script type="text/javascript" src="../../library/textformat.js?v=<?php echo $v_js_includes; ?>"></script>
-<script type="text/javascript" src="../../library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-7-2/index.js"></script>
-<?php include_once("{$GLOBALS['srcdir']}/ajax/payment_ajax_jav.inc.php"); ?>
-<script type="text/javascript" src="../../library/js/common.js?v=<?php echo $v_js_includes; ?>"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker-2-5-4/build/jquery.datetimepicker.full.min.js"></script>
-
-<script language='JavaScript'>
- var mypcc = '1';
-</script>
-<script type="text/javascript" language="javascript" >
-function Validate()
- {
-  if(document.getElementById('uploadedfile').value=='')
-   {
-    alert("<?php echo htmlspecialchars(xl('Please Choose a file'), ENT_QUOTES) ?>");
-    return false;
-   }
-  if(document.getElementById('hidden_type_code').value=='')
-   {
-    alert("<?php echo htmlspecialchars(xl('Select Insurance, by typing'), ENT_QUOTES) ?>");
-    document.getElementById('type_code').focus();
-    return false;
-   }
-  if(document.getElementById('hidden_type_code').value!=document.getElementById('div_insurance_or_patient').innerHTML)
-   {
-    alert("<?php echo htmlspecialchars(xl('Take Insurance, from Drop Down'), ENT_QUOTES) ?>");
-    document.getElementById('type_code').focus();
-    return false;
-   }
-    top.restoreSession();
-    document.forms[0].submit();
- }
-function OnloadAction()
- {//Displays message after upload action,and popups the details.
-  after_value=document.getElementById('after_value').value;
-  if(after_value!='')
-   {
-    alert(after_value);
-   }
-    <?php
-    if ($_FILES['form_erafile']['size']) {
+    <?php Header::setupHeader(['datetime-picker']);?>
+    <script type="text/javascript" src="../../library/js/common.js?v=<?php echo $v_js_includes; ?>"></script>
```

Comment:
bring the common.js include into Header call

Reply:
Done